gradle.properties

```diff
@@ -6,4 +6,4 @@ slf4jVersion=1.7.30
 testngVersion=7.4.0
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.2.0
+ballerinaLangVersion=2201.4.0-20221116-082400-3eca35ee
```

Comment:
Woah, The module version and the distribution version in Ballerina.toml needs to be changed

Reply:
@TharmiganK FYA

Reply:
Fixed via https://github.com/ballerina-platform/module-ballerina-constraint/pull/35